lib/jekyll/feed_meta_tag.rb

```diff
@@ -26,6 +26,14 @@ def url
         config["github"]["url"]
       end
     end
+
+    def title
```

Comment:
I am trying to avoid output of `title=""` if neither is specified.

Am I overthinking this?